### DISCUSSION TITLE
React accordion section - sorting in created order not alphabetical order

### DISCUSSION THREAD

**Discussion by gonfishin** [pnp/sp-dev-fx-webparts#1926] [Q&A] [answered]
Hi, firstly I would like to say thanks to all the contributors for their work on this webpart.

@hugoabernier, I installed the webpart and find that it's listing the entries by the order that the items were created in the referenced list.  The SharePoint list is sorted alphabetically in the default view.

Is there a way to have the accordion webpart display current and future entries in alphabetical order?

![faq](https://user-images.githubusercontent.com/85678373/121508328-d2426000-ca28-11eb-9c24-04be612de57c.JPG)

**Comment by AriGunawan** [comment, marked as answer]
Hi @gonfishin,
I don't find any configuration for ordering the items in that web part.

I think the only way to make it display in alphabetical order is by updating the code. 
You can try to update the [ReactAccordion.tsx](https://github.com/pnp/sp-dev-fx-webparts/blob/main/samples/react-accordion-section/src/webparts/reactAccordion/components/ReactAccordion.tsx) file to get the list items ordered by the title:
```diff
  private getListItems(): void {
    if(typeof this.props.listId !== "undefined" && this.props.listId.length > 0) {
-      sp.web.lists.getById(this.props.listId).items.select("Title","Content").get()
+      sp.web.lists.getById(this.props.listId).items.select("Title","Content").orderBy("Title").get()
        .then((results: Array<any>) => {
          this.setState({
            items: results
          });
        })
        .catch((error:any) => {
          console.log("Failed to get list items!");
          console.log(error);
        });
    }
  }
```

**Reply by gonfishin** [reply]
@AriGunawan thank you for taking your time to answer my question.  Your solution worked perfectly.   👍
